libs/state/jest.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   displayName: 'state',
   preset: '../../jest.preset.js',
   coverageReporters: ['lcov', 'cobertura'],
-  coverageDirectory: '../../docs/test-coverage/state',
+  coverageDirectory: '../../dist/apps/docs/test-coverage/state',
```

Comment:
The build output directory for the `docs` project is now `dist/apps/docs` instead of the raw files in `docs`.